gtsfm/scene_optimizer.py

```diff
@@ -216,6 +219,11 @@ def _filter_dict_keys(dict: Dict[Any, Any], ref_dict: Dict[Any,Any]) -> Dict[Any
         # Save metrics to JSON and generate HTML report.
         auxiliary_graph_list.extend(save_metrics_reports(metrics_graph_list))
 
+        # Modify BA input and BA output to have point clouds and frustums aligned with x,y,z axes.
+        aligned_pose_graphs = dask.delayed(align_estimated_gtsfm_data)(ba_input_graph, ba_output_graph)
+        ba_input_graph = aligned_pose_graphs[0]
+        ba_output_graph = aligned_pose_graphs[1]
```

Comment:
can you post an example of the output matplotlib plots with this code change? we need to apply the transformation to align to orthogonal axes to the ground truth here, otherwise the plotting code will look incorrect.

Reply:
this code will be cleaner with 
```python
ba_input_graph, ba_output_graph = ask.delayed(align_estimated_gtsfm_data, nout=2)(ba_input_graph, ba_output_graph)
```

Reply:
Slacked you the matplotlib plots of all the camera poses. Just added this line under `align_estimated_gtsfm_data()` method:
`gt_pose_graph = [walignedTw.transformFrom(wTi) for wTi in gt_pose_graph]`

Reply:
thanks!